### ISSUE TITLE
`pdfkit` errors

### ISSUE THREAD

**Issue by rauchg** [zeit/ncc#2]
ncc version: 0.22

Source:

```js
const fs = require("fs");
const PDFDocument = require("pdfkit");
const doc = new PDFDocument();
doc.fontSize(15).text("Hi there", 50, 50);
doc.pipe(fs.createWriteStream("out.pdf"));
doc.end();
```

Expected:

No output, working pdf:

```
▲  play-ncc/ node index.js
▲  play-ncc/
```

![image](https://user-images.githubusercontent.com/13041/48390891-f98df080-e6b8-11e8-9d81-3f0ae6b0fe93.png)

Received:

```
▲  play-ncc/ node out.js
internal/modules/cjs/loader.js:589
    throw err;
    ^

Error: Cannot find module 'iconv-lite'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:587:15)
    at Function.Module._load (internal/modules/cjs/loader.js:513:25)
    at Module.require (internal/modules/cjs/loader.js:643:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/Users/rauchg/Projects/play-ncc/out.js:1:270)
    at Module._compile (internal/modules/cjs/loader.js:707:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:718:10)
    at Module.load (internal/modules/cjs/loader.js:605:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:544:12)
    at Function.Module._load (internal/modules/cjs/loader.js:536:3)
```

Compilation output:

```
▲  play-ncc/ ncc index.js -o out.js
'iconv-lite' is imported by node_modules/fontkit/index.js, but could not be resolved – treating it as an external dependency
'iconv-lite' is imported by node_modules/restructure/src/DecodeStream.js, but could not be resolved – treating it as an external dependency
'iconv-lite' is imported by node_modules/restructure/src/EncodeStream.js, but could not be resolved – treating it as an external dependency
'iconv-lite' is imported by commonjs-external:iconv-lite, but could not be resolved – treating it as an external dependency
Circular dependency: node_modules/pdfkit/js/object.js -> node_modules/pdfkit/js/reference.js -> node_modules/pdfkit/js/object.js
Circular dependency: node_modules/pdfkit/js/object.js -> node_modules/pdfkit/js/reference.js -> commonjs-proxy:/Users/rauchg/Projects/play-ncc/node_modules/pdfkit/js/object.js -> node_modules/pdfkit/js/object.js
Circular dependency: node_modules/pdfkit/js/font.js -> node_modules/pdfkit/js/font/standard.js -> node_modules/pdfkit/js/font.js
Circular dependency: node_modules/pdfkit/js/font.js -> node_modules/pdfkit/js/font/standard.js -> commonjs-proxy:/Users/rauchg/Projects/play-ncc/node_modules/pdfkit/js/font.js -> node_modules/pdfkit/js/font.js
Circular dependency: node_modules/pdfkit/js/font.js -> node_modules/pdfkit/js/font/embedded.js -> node_modules/pdfkit/js/font.js
```

**Comment by guybedford** [comment]
This sounds like the circular reference issue as well (possible dupe of #10 depending on how we solve these).

**Comment by guybedford** [comment]
Code in question here is exactly - 

```js
  var AFMFont, PDFFont, StandardFont, fs$$1,
    extend = function(child, parent) { for (var key in parent) { if (hasProp.call(parent, key)) child[key] = parent[key]; } function ctor() { this.constructor = child; } ctor.prototype = parent.prototype; child.prototype = new ctor(); child.__super__ = parent.prototype; return child; },
    hasProp = {}.hasOwnProperty;
```

so it does seem like a cycle bug to me. Adding cycle label.

**Comment by rauchg** [comment]
Fixed by #39
